db/warehouse/migrate/20230301170853_remove_custom_assessments.rb

```diff
@@ -0,0 +1,8 @@
+class RemoveCustomAssessments < ActiveRecord::Migration[6.1]
+  def up
+    # Remove any custom assessments from the HUD CE Assessments table
+    Hmis::Hud::Assessment.joins(:assessment_detail).destroy_all
+    Hmis::Hud::Assessment.where(AssessmentType: 999).destroy_all
+    Hmis::Wip.where(source_type: 'Hmis::Hud::Assessment').destroy_all
```

Comment:
Just removing old custom assessments rather than migrating, because we should do this anyway for local environments since the values/hudValues shapes changed recently